### DISCUSSION TITLE
Canva Pro

### DISCUSSION THREAD

**Discussion by Keirra23** [community/community#29180] [GitHub Education]
I am new to the Github community. I don't see the canva pro benefits anywhere

**Comment by pasindu651** [comment]
Hi @Keirra23, 
Unfortunately, the Canva Pro benefit is no longer available and is no longer offered by the Github Student Dev Pack. However, those who already have the benefit will continue to until the end of the 12-month period.

**Reply by pasindu651** [reply]
> Hi @Keirra23, Unfortunately, the Canva Pro benefit is no longer available and is no longer offered by the Github Student Dev Pack. However, those who already have the benefit will continue to until the end of the 12-month period.

Hi @Keirra23 unfortunately it's not available anymore.

**Comment by Keirra23** [comment]
I signed up already and was approved. I don’t see the CANVA pro option
